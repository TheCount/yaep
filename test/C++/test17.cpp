#include <stdlib.h>

#include"common.h"

static const char *input = "a+a";

static const char *description =
"\n"
"E : 'a' E\n"
"  | E '+'\n"
"  ;\n"
  ;

int
main (int argc, char **argv)
{
  test_complex_parse (input, description, 0, 0, 0, 0, argc, argv);
  exit (0);
}
