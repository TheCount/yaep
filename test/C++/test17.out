nonterm `E' does not derive any term string
